src/main/java/com/cglx/Calculator.java

```diff
@@ -36,6 +36,30 @@ public int square(int a) {
 		int result = a * a;
 		return result;
 	}
+	
+	public int remainder(int a, int b) {
+		int result = a % b;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "result". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS1488)